### PR TITLE
created bottom part of frontpage

### PR DESCRIPTION
**Pull request by kbojarowska**

### CODE REVIEW

File: skin-resque-frontend/src/pages/FrontPage/FrontPage.js [hunk 1]
```diff
@@ -1,8 +1,55 @@
...
+      "description": `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+      ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+      ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit
+      in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, 
+      sunt in culpa qui officia deserunt mollit anim id est laborum.`,
+      "photo": '/images/rosemary.jpg',
+    }
+  ]
+
+  const [showIngredient, setShowIngredient] = useState(ingredients[0])
```
Comment (MrSpaceMan1) [inline skin-resque-frontend/src/pages/FrontPage/FrontPage.js:34]: Wouldn't simple index state be easier and quicker to do?

File: skin-resque-frontend/src/pages/FrontPage/FrontPage.js [hunk 2]
```diff
@@ -1,8 +1,55 @@
 import './FrontPage.scss';
 import photo from '../../resources/front-page-skin-photo.jpg';
-import Button from '../../components/Button/Button'
+import Button from '../../components/Button/Button';
+import Text from '../../components/Text/Text';
+import Heading from '../../components/Heading/Heading';
+import Arrow from '../../components/Arrow/Arrow';
+import { useState } from 'react';
```
Comment (MrSpaceMan1) [inline skin-resque-frontend/src/pages/FrontPage/FrontPage.js:7]: Module imports should occur first

File: skin-resque-frontend/src/pages/FrontPage/FrontPage.js [hunk 3]
```diff
@@ -23,7 +70,21 @@ function FrontPage() {
         </div>
       </div>
       <img src={photo} className='skinPhoto'></img>
-      <div className='white'></div>
+      <div className='white-info'>
```
Comment (MrSpaceMan1) [inline skin-resque-frontend/src/pages/FrontPage/FrontPage.js:73]: Will this sort of structure used elsewhere? If so you should consider making it standalone component

**Review by MrSpaceMan1 — DISMISSED** [review]

File: skin-resque-frontend/src/pages/FrontPage/FrontPage.js (hunk 1 above)
Comment (kbojarowska) [reply skin-resque-frontend/src/pages/FrontPage/FrontPage.js:34]: I am not sure about this

**Review by dpionk — APPROVED** [review]
LGTM

**Pull request merged**